_dev/src/scss/components/_index.scss

```diff
@@ -14,4 +14,6 @@
 @use "privacy";
 @use "progress";
 @use "radio-card";
+@use "scrollbar";
```

Comment:
`scrollbar.scss` has already been removed in the virtual scroll PR, so the @use directive needs to be removed; otherwise, we encounter a compilation error.
```suggestion
```